### PR TITLE
Add ports section to CRM

### PR DESCRIPTION
**Pull request by eyakubovich**
Need a way to specify how the container ports will get mapped
to the host ports.

### CODE REVIEW

**Comment by jonboulle** [comment]
Per our OOB discussion, this seems reasonable to me. You will need to update the schema code.

**Comment by eyakubovich** [comment]
@jonboulle oh yeah, forgot about the code.

**Comment by philips** [comment]
lgtm

File: SPEC.md [hunk 1]
```diff
@@ -856,3 +862,6 @@ JSON Schema for the Container Runtime Manifest (container manifest), conforming
     * **readOnly** (boolean, optional if **kind** is "host") whether or not the volume should be mounted read only.
 * **isolators** (list of objects, optional) list of isolators that will apply to all apps in this container. Each object has two key value pairs: **name** is restricted to the AC Name formatting and **value** can be a freeform string)
 * **annotations** (list of objects, optional) arbitrary metadata the executor should make available to applications via the metadata service. Objects must contain two key-value pairs: **name** is restricted to the [AC Name](#ac-name-type) formatting and **value** is an arbitrary string). Annotation names must be unique within the list.
+* **ports** (list of objects, optional) list of ports that will be exposed on the host.
+    * **name** (string, required) name the port in the image manifest that should be exposed on the host (restricted to the AC Name formatting).
+    * **hostPort** (integer, required) the port number on the host that will be mapped to the container port.
```
Comment (alban) [inline SPEC.md]: Does it mean the port on the host needs to be the same as the port in the container?

Comment (eyakubovich) [inline SPEC.md]: @alban not at all. The "name" references a port defined in the image manifest which contains (protocol, container port) information. hostPort actually allows for the host port to be distinct from the container port.

Comment (alban) [inline SPEC.md]: Oh ok, I understand now. Thanks for the explanation.

**Comment by cdaylward** [comment]
LGTM

**Pull request merged**